### PR TITLE
cloud: refine the steps for configuring the changefeed target

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 1]
```diff
@@ -66,14 +66,14 @@ For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources
 ## Step 2. Configure the changefeed target
 
 1. Under **Brokers Configuration**, fill in your Kafka brokers endpoints. You can use commas `,` to separate multiple endpoints.
-2. Select your Kafka version. If you do not know that, use Kafka V2.
-3. Select a desired compression type for the data in this changefeed.
-4. Enable the **TLS Encryption** option if your Kafka has enabled TLS encryption and you want to use TLS encryption for the Kafka connection.
-5. Select the **Authentication** option according to your Kafka authentication configuration.
+2. Select the **Authentication** option according to your Kafka authentication configuration.
 
     - If your Kafka does not require authentication, keep the default option **DISABLE**.
```
Comment (lilin90) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:71]:
```suggestion
    - If your Kafka does not require authentication, keep the default option **Disable**.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 2]
```diff
@@ -66,14 +66,14 @@ For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources
 ## Step 2. Configure the changefeed target
 
 1. Under **Brokers Configuration**, fill in your Kafka brokers endpoints. You can use commas `,` to separate multiple endpoints.
-2. Select your Kafka version. If you do not know that, use Kafka V2.
-3. Select a desired compression type for the data in this changefeed.
-4. Enable the **TLS Encryption** option if your Kafka has enabled TLS encryption and you want to use TLS encryption for the Kafka connection.
-5. Select the **Authentication** option according to your Kafka authentication configuration.
+2. Select the **Authentication** option according to your Kafka authentication configuration.
```
Comment (lilin90) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:69]:
```suggestion
2. Select the authentication option according to your Kafka authentication configuration.
```

Comment (qiancai) [reply tidb-cloud/changefeed-sink-to-apache-kafka.md:69]: Select an authentication option

**Review by lilin90 — COMMENTED** [review]
/lgtm

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 3]
```diff
@@ -66,14 +66,14 @@ For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources
 ## Step 2. Configure the changefeed target
 
 1. Under **Brokers Configuration**, fill in your Kafka brokers endpoints. You can use commas `,` to separate multiple endpoints.
-2. Select your Kafka version. If you do not know that, use Kafka V2.
-3. Select a desired compression type for the data in this changefeed.
-4. Enable the **TLS Encryption** option if your Kafka has enabled TLS encryption and you want to use TLS encryption for the Kafka connection.
-5. Select the **Authentication** option according to your Kafka authentication configuration.
+2. Select the authentication option according to your Kafka authentication configuration.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:69]:
```suggestion
2. Select an authentication option according to your Kafka authentication configuration.
```

File: tidb-cloud/tidb-cloud-billing-dm.md [hunk 4]
```diff
@@ -11,16 +11,21 @@ This document describes the billing for Data Migration in TiDB Cloud.
 
 TiDB Cloud measures the capacity of Data Migration in Replication Capacity Units (RCUs). When you create a Data Migration job, you can select an appropriate specification. The higher the RCU, the better the migration performance. You will be charged for these Data Migration RCUs.
 
-The following table lists the specifications and corresponding performances for Data Migration.
+The following table lists the specifications, corresponding performances, and the maximum number of tables for Data Migration.
```
Comment (Frank945946) [inline tidb-cloud/tidb-cloud-billing-dm.md:14]:
```suggestion
The following table lists the corresponding performances and the maximum number of tables that can be migrated for each Data Migration specification.
```

File: tidb-cloud/tidb-cloud-billing-dm.md [hunk 5]
```diff
@@ -11,16 +11,21 @@ This document describes the billing for Data Migration in TiDB Cloud.
...
+| 4 RCUs  | 35 MiB/s | 20,000 rows/s | 10000 |
+| 8 RCUs  | 40 MiB/s | 40,000 rows/s | 30000 |
+| 16 RCUs | 45 MiB/s | 80,000 rows/s | 60000 |
 
-Note that all the performance values in this table are maximum performances. It is assumed that there are no performance, network bandwidth, or other bottlenecks in the upstream and downstream databases. The performance values are for reference only and might vary in different scenarios.
+For more information about the prices of Data Migration RCUs, see [Data Migration Cost](https://www.pingcap.com/tidb-dedicated-pricing-details/#dm-cost).
+
+> **Note:**
+>
+> - If the number of tables exceeds the maximum number of tables, the Data Migration job can still run, but the job might become unstable or even fail.
```
Comment (Frank945946) [inline tidb-cloud/tidb-cloud-billing-dm.md:27]:
```suggestion
> - If the number of tables to be migrated exceeds the maximum number of tables, the Data Migration job may still run, but the job might become unstable or even fail.
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-billing-dm.md [hunk 6]
```diff
@@ -11,16 +11,21 @@ This document describes the billing for Data Migration in TiDB Cloud.
 
 TiDB Cloud measures the capacity of Data Migration in Replication Capacity Units (RCUs). When you create a Data Migration job, you can select an appropriate specification. The higher the RCU, the better the migration performance. You will be charged for these Data Migration RCUs.
 
-The following table lists the specifications and corresponding performances for Data Migration.
+The following table lists the corresponding performances and the maximum number of tables that can be migrated by each Data Migration specification.
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-billing-dm.md:14]:
```suggestion
The following table lists the corresponding performance and the maximum number of tables that each Data Migration specification can migrate.
```

File: tidb-cloud/tidb-cloud-billing-dm.md [hunk 7]
```diff
@@ -11,16 +11,21 @@ This document describes the billing for Data Migration in TiDB Cloud.
...
+| 4 RCUs  | 35 MiB/s | 20,000 rows/s | 10000 |
+| 8 RCUs  | 40 MiB/s | 40,000 rows/s | 30000 |
+| 16 RCUs | 45 MiB/s | 80,000 rows/s | 60000 |
 
-Note that all the performance values in this table are maximum performances. It is assumed that there are no performance, network bandwidth, or other bottlenecks in the upstream and downstream databases. The performance values are for reference only and might vary in different scenarios.
+For more information about the prices of Data Migration RCUs, see [Data Migration Cost](https://www.pingcap.com/tidb-dedicated-pricing-details/#dm-cost).
+
+> **Note:**
+>
+> - If the number of tables to be migrated exceeds the maximum number of tables, the Data Migration job might can still run, but the job might become unstable or even fail.
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-billing-dm.md:27]:
```suggestion
> - If the number of tables to be migrated exceeds the maximum number of tables, the Data Migration job might still run, but the job could become unstable or even fail.
```

File: tidb-cloud/tidb-cloud-billing-dm.md [hunk 8]
```diff
@@ -11,16 +11,21 @@ This document describes the billing for Data Migration in TiDB Cloud.
...
+> - All the performance values in this table are maximum performances. It is assumed that there are no performance, network bandwidth, or other bottlenecks in the upstream and downstream databases. The performance values are for reference only and might vary in different scenarios.
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-billing-dm.md:28]:
```suggestion
> - All the performance values in this table are maximum and optimal ones. It is assumed that there are no performance, network bandwidth, or other bottlenecks in the upstream and downstream databases. The performance values are for reference only and might vary in different scenarios.
```

**Pull request merged**